addons/website_sale_collect/static/src/js/checkout.js

```diff
@@ -51,11 +51,14 @@ publicWidget.registry.WebsiteSaleCheckout.include({
         }
         // TODO: move logic below to `_isDeliveryMethodReady` override on master
         const checkedRadio = this.el.querySelector('input[name="o_delivery_radio"]:checked');
-        const deliveryContainer = this._getDeliveryMethodContainer(checkedRadio);
-        const hasWarning = (
-            checkedRadio.dataset.deliveryType === 'in_store'
-            && deliveryContainer.querySelector('[name="unavailable_products_warning"]')
-        );
```

Comment:
Why not just that?
```suggestion
        const hasWarning = (
            checkedRadio?.dataset.deliveryType === 'in_store'
            && deliveryContainer?.querySelector('[name="unavailable_products_warning"]')
        );
```

Reply:
I think it's clear enough that we branch out if the radio is not found.